netstandard/tools/targets/NETStandard.Library.targets

```diff
@@ -137,4 +139,6 @@
       <Output TaskParameter="TouchedFiles" ItemName="FileWrites" />
     </Touch>
   </Target>
+
+  <Import Project="NETStandard.Library.Trimming.targets" Condition="'$(_SupportsTrimming)' == 'true' AND '$(TrimUnusedDependencies)' == 'true'" />
```

Comment:
Given you set this to true above what cases do you expect to override and set that to false? Also nit I casing is different for your "s" in _SupportsTrimming in the two places.